config/config.go

```diff
@@ -68,6 +68,7 @@ func (config *Config) parseFlags() {
 	flag.StringVar(&config.Consul.SslCaCert, "consul-ssl-ca-cert", "", "Path to a CA certificate file, containing one or more CA certificates to use to validate the certificate sent by the Consul server to us")
 	flag.StringVar(&config.Consul.Token, "consul-token", "", "The Consul ACL token")
 	flag.StringVar(&config.Consul.Tag, "consul-tag", "marathon", "Common tag name added to every service registered in Consul, should be unique for every Marathon-cluster connected to Consul")
+	flag.DurationVar(&config.Consul.Timeout, "consul-timeout", time.Second, "Max request duration")
```

Comment:
I would suggest a little change: "Time limit for requests made by the Consul HTTP client. A timeout of zero means no timeout."